src/main/java/net/fabricmc/mappingpoet/Main.java

```diff
@@ -152,7 +151,9 @@ private static void writeClass(MappingsStore mappings, ClassNode classNode, Map<
 			while (lastSearch != -1) {
 				lastSearch = name.lastIndexOf('$', lastSearch - 1);
 				if (isDigit(name.charAt(lastSearch + 1))) // names starting with digit is illegal java
+				{
```

Comment:
I would've put the `{` before the `//`. Doesn't really matter though.

Reply:
Will fix this code style issue (move comment above `if` imo is the best)